src/nunavut/lang/cpp/__init__.py

```diff
@@ -8,25 +8,24 @@
     module will be available in the template's global namespace as ``cpp``.
 """
 
-import functools
 import fractions
+import functools
```

Comment:
in the CPP changes we have both support file generation changes but we also move away from `std::vector` as the default variable-length array and, instead, use our built-in VariableLengthArray template. More on the design of this below.

The intent is to allow, if not encourage, users to provide their own VariableLengthArray implementation or their own allocator for the built-in version. That will be added in a subsequent change.